CadRevealFbxProvider/BatchUtils/ScaffoldOptimizer.cs

```diff
@@ -0,0 +1,116 @@
+namespace CadRevealFbxProvider.BatchUtils;
+
+using System.Linq;
+using CadRevealComposer;
+using CadRevealComposer.Primitives;
+using CadRevealComposer.Tessellation;
+using ScaffoldPartOptimizers;
+
+public static class ScaffoldOptimizer
+{
+    public static void AddPartOptimizer(ScaffoldPartOptimizer optimizer)
+    {
+        PartOptimizers.Add(optimizer);
+    }
```

Comment:
Consider making the class non-static if we have mutable state. Or add a AddPartOptimizerTESTONLY suffix